connectivity/check/action.go

```diff
@@ -161,21 +161,21 @@ func (a *Action) ExecInPod(ctx context.Context, cmd []string) {
 	cmdName := cmd[0]
 	cmdStr := strings.Join(cmd, " ")
 
-	if stderr.Len() > 0 {
-		a.test.Debugf("%s stderr: %s", cmdName, stderr.String())
-	} else if stdout.Len() > 0 {
-		a.test.Debugf("%s stdout: %s", cmdName, stdout.String())
-	}
-
 	if err != nil || stderr.Len() > 0 {
 		if a.shouldSucceed() {
+			// Command failed unexpectedly, display stderr.
 			a.Failf("command %q failed: %s", cmdStr, err)
+			a.test.Infof("%s stderr:\n%s\n", cmdName, strings.TrimSpace(stderr.String()))
```

Comment:
In general, I personally prefer to not include newlines in formatted output, but instead just issue another `Infof`. It makes it easier to read the string being printed from the code point of view. WDYT?

Reply:
Oh, totally! stdout/err is usually multi-line, so would probably use a `Log()` for the second call. Thanks!